src/sentry/runner/commands/configoptions.py

```diff
@@ -0,0 +1,229 @@
+import click
+
+from sentry.runner.decorators import configuration
+
+
+@click.group()
+def configoptions():
+    "Manages Sentry options."
+
+
+@configoptions.command()
+@configuration
+def list():
+    "Fetches all options."
+    from sentry import options
+
+    for opt in options.all():
+        # click.echo(f"{opt.name} ({opt.type}) = {options.get(opt.name)}")
+        # click.echo(type(opt))
+        if can_change(opt.name):
+            click.echo(f"{opt.name}: {options.get(opt.name)}")
+
+
+@configoptions.command()
+@click.argument("filename", required=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    default=False,
+    required=False,
+    help="Output exactly what changes would be made and in which order.",
+)
+@configuration
+def patch(filename: str, dryrun: bool):
+    "Updates, gets, and deletes options that are each subsectioned in the given file."
+    import yaml
+
+    if dryrun:
+        click.echo("Dryrun flag on. ")
+    with open(filename) as stream:
+        configmap_data = yaml.safe_load(stream)
+        data = configmap_data.get("data", {}).get("options-patch.yaml", "")
+
+        keysToFetch = data.get("fetch", {})
+        keysToUpdate = data.get("update", {})
+        keysToDelete = data.get("delete", {})
+
+        for key in keysToFetch:
+            _get(key, dryrun)
+
+        for key, val in keysToUpdate.items():
+            _set(key, val, dryrun)
+
+        for key in keysToDelete:
+            _delete(key, dryrun)
+
+
+@configoptions.command()
+@click.argument("filename", required=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    default=False,
+    required=False,
+    help="Output exactly what changes would be made and in which order.",
+)
+@configuration
+def strict(filename: str, dryrun: bool):
+    "Deletes everything not in the uploaded file, and applies all of the changes in the file."
+    import yaml
+
+    if dryrun:
+        click.echo("Dryrun flag on. ")
+
+    with open(filename) as stream:
+        configmap_data = yaml.safe_load(stream)
+        data = configmap_data.get("data", {}).get("options-strict.yaml", "")
+
+        for key in TRACKED:
+            if key not in data.keys():
+                _delete(key)
+
+        for key, val in data.items():
+            if key in TRACKED:
+                _set(key, val, dryrun)
+
+
+@configoptions.command()
+@click.argument("key", required=True)
+@click.option(
+    "--dryrun",
+    is_flag=True,
+    default=False,
+    required=False,
+    help="Output exactly what changes would be made and in which order.",
+)
+@configuration
+def get(key: str, dryrun: bool = False) -> str:
+    "Get a configuration option."
+    return _get(key, dryrun)
+
+
+def _get(key: str, dryrun: bool = False) -> str:
+    from sentry import options
+    from sentry.options.manager import UnknownOption
```

Comment:
After offline discussion, actually you are right, since the helper function is in the same module as the entry point you would have to re-import everything, which is unfortunate. 
Could you verify whether you can actually move these import at the top of the module ? If that does not work then we are good with this .